docs/4-Commands/0-kool.md

```diff
@@ -21,7 +21,6 @@ Complete documentation is available at https://kool.dev/docs
 * [kool docker](kool-docker.md)	 - Creates a new container and runs the command in it.
 * [kool exec](kool-exec.md)	 - Execute a command within a running service container
 * [kool info](kool-info.md)	 - Prints out information about kool setup (like environment variables)
-* [kool init](kool-init.md)	 - [DEPRECATED] Proxies preset command
```

Comment:
this is being removed only on `2.0`... kool init is proxing kool preset and I guess the script to build de docs isn't handling it...

Reply:
not sure why this got removed, `make-docs ` made it... did you in the past manually returned this line? I guess deprecated commands/hidden commands do not show up by default in the extracted docs?

Reply:
@fabriciojs have no idea why `make-docs` removes it.

Reply:
@danielsuguimoto just confirmed that if we remove the `initCmd.Deprecated` message from the `init.go` command, then `make-docs` will include it. `docs.go`, for the main command docs, do not include by default Non-available commands.

![image](https://user-images.githubusercontent.com/1116377/105866882-36f9a280-5fd3-11eb-8802-a7d1f6565b0e.png)

So since this is a builtin behavior, I am OK with sticking with it as well. The command is still available, and we do not show the deprecated commands in the list, just like `kool -h` do not:

```
➜  kool git:(docker-compose) kool
An easy and robust software development environment
tool helping you from project creation until deployment.
Complete documentation is available at https://kool.dev/docs

Usage:
  kool [command]

Available Commands:
  create      Create a new project using preset
  deploy      Deploys your application using Kool Dev
  docker      Creates a new container and runs the command in it.
  exec        Execute a command within a running service container
  help        Help about any command
  info        Prints out information about kool setup (like environment variables)
  logs        Displays log output from services.
  preset      Initialize kool preset in the current working directory. If no preset argument is specified you will be prompted to pick among the existing options.
  restart     Restart containers - the same as stop followed by start.
  run         Runs a custom command defined at kool.yaml in the working directory or in the kool folder of the user's home directory
  self-update Update kool to latest version
  start       Start the specified Kool environment containers. If no service is specified, start all.
  status      Shows the status for containers
  stop        Stop all running containers started with 'kool start' command

Flags:
  -h, --help      help for kool
      --verbose   increases output verbosity
  -v, --version   version for kool

Use "kool [command] --help" for more information about a command.
```